workers/unity/Assets/Playground/Scripts/OneTimeInitialisation.cs

```diff
@@ -18,38 +18,11 @@ private static void Init()
             }
 
             initialized = true;
-            SetupInjectionHooks();
-            PlayerLoopManager.RegisterDomainUnload(DomainUnloadShutdown, 1000);
+            CoreInitialization.SetupInjectionHooks();
+            PlayerLoopManager.RegisterDomainUnload(CoreInitialization.DomainUnloadShutdown, 1000);
```

Comment:
I think this should be part of the new class as well, so the users would call only one method

Reply:
"this" being `RegisterDomainUnload`

Reply:
I kept it here intentionally so that users can define their own `RegisterDomainUnload` if necessary. They might want to do more clean up than we provide

Reply:
Agreed, sounds sensible